### PR TITLE
Dev maria teo portfolio section finished

### PR DESCRIPTION
**Pull request by MariaTeo**

### CODE REVIEW

**Review by samuray90 — APPROVED** [review]

**Review by samuray90 — APPROVED** [review]
👏

File: src/_component/portfolio/portfolio.pug [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    each project in data.portfolio.project1 
+      .recent__projects
+        .recent__projects-text=project.title
+        .recent__projects-skills=project.skills
+        .recent__projects-skills=project.link
+        img(src=`${project.image}`, alt="avatar").project__image
+      .buttons
+        a(href=`${project.linkIcon}` target="_blank")
+          i.icon__earth 
+        a(href=`${project.linkIcon}` target="_blank")
```
Comment (marianzburlea) [inline src/_component/portfolio/portfolio.pug]: you don't need ticks to evaluate one single value
```a(href=`${project.linkIcon}` target="_blank")```
to
```a(href=project.linkIcon target="_blank")```

File: src/_component/portfolio/portfolio.pug [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          i.icon__github
+.column(span="row" height="10")
+.column(span="row")
+  .data__stat=data.portfolio.project2.title
+    each project in data.portfolio.project2.data
+      .old__projects
+        .recent__projects-text=project.title
+        .recent__projects-skills=project.skills
+        .recent__projects-skills=project.link
+        img(src=`${project.image}`, alt="avatar").project__image
```
Comment (marianzburlea) [inline src/_component/portfolio/portfolio.pug]: same as above

File: src/_data/portfolio.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "title":"Old",
+    "data":
+    [
+      {
+        "title":"CodeTap Academy",
+        "skills":"HTML, CSS, Git, JavaScript, CEO, React",
+        "link": "https://codetap.academy",
+        "image":"asset/image/company-logo/romaris.png",
+        "linkIcon":"https://codetap.academy"
+
```
Comment (marianzburlea) [inline src/_data/portfolio.json]: leave spaces after colon:
```"skills":"HTML, CSS, Git, JavaScript, CEO, React",```
to
```"skills": "HTML, CSS, Git, JavaScript, CEO, React",```

**Review by marianzburlea — CHANGES_REQUESTED** [review]
A few changes

**Review by marianzburlea — APPROVED** [review]
💯

**Review by psyhedeliq — APPROVED** [review]
Nice! 🚀 👏

**Pull request merged**